docs/src/app/components/pages/components/select-fields.jsx

```diff
@@ -173,7 +173,7 @@ const SelectFieldsPage = React.createClass({
           },
           {
             name: 'onChange',
-            header: 'function(name, event)',
+            header: 'function(event, name)',
```

Comment:
that's not the name, that's the `selectedIndex` as for the DropDownMenu component.